Analytics-API/MQTT.md

```diff
@@ -0,0 +1,116 @@
+---
+id: mqtt-integration
+title: MQTT Integration
+description: Guide on integrating PlaceOS with MQTT messaging
+---
+<!-- # MQTT -->
+
+<!-- images in old file, need rehosting https://docs.google.com/document/d/1gBZD296sF0cZXYyRKrRjn-wi5c408Yg1189S-DV9JgE/edit# -->
+
+PlaceOS supports publishing module state information via MQTT. 
```

Comment:
```suggestion
PlaceOS supports publishing module state information via <abbr title="Message Queuing Telemetry Transpot">MQTT</abbr>.
```

Alternatively, while it's not part of the markdown, there is a fairly clean, common syntax that seem to be [used elsewhere](https://talk.commonmark.org/t/abbreviations-and-acronyms/890):

```markdown
This is a sentence with an acronym: MQTT.

*[MQTT]: Message Queuing Telemetry Transport
```

Can add support via https://github.com/zestedesavoir/zmarkdown/tree/HEAD/packages/remark-abbr.

Reply:
Just coming back to this, I think given our other choices for relatively-universal-readability, I think just going with manual `<abbr>` tags on the first instance of an acronym on a page is our best bet.